ui/src/app/base/components/FormikFormButtons/FormikFormButtons.tsx

```diff
@@ -101,32 +93,27 @@ export const FormikFormButtons = <V,>({
 
   return (
     <>
-      {buttonsBordered && <hr />}
       <div
-        className={classNames(buttonsClassName, {
-          "u-flex--between":
-            !inline && (buttonsAlign === "right" || showHelpLink),
+        className={classNames("formik-form-buttons", buttonsClassName, {
+          "is-bordered": buttonsBordered,
+          "is-inline": inline,
         })}
+        data-test="buttons-wrapper"
       >
-        <p className="u-no-margin--bottom u-no-max-width">
-          {showHelpLink ? (
-            <Link
-              external
-              href={buttonsHelpLink}
-              rel="noopener noreferrer"
-              target="_blank"
-            >
-              {buttonsHelpLabel}
-            </Link>
-          ) : null}
-        </p>
-        <div>
+        {(buttonsHelp || buttonsAlign === "right") && (
```

Comment:
You already handle for button alignment within the class assignment. Do you need it here too?

Reply:
I did originally, but I've address it [in the scss](https://github.com/canonical-web-and-design/maas-ui/pull/2949/files#diff-206c2af5347212109733fea197ea1c16e92941c111c2b77582348f7df606adc5R23).